src/webapp/pages/incident-action-plan/useIncidentActionPlan.ts

```diff
@@ -129,6 +131,19 @@ export function useIncidentActionPlan(id: Id) {
         }
     }, [changeCurrentEventTracker, currentEventTracker, incidentActionExists, incidentActionPlan]);
 
+    useEffect(() => {
+        compositionRoot.userGroup.getByCode.execute(RTSL_ZEBRA_INCIDENTMANAGER).run(
+            userGroup => {
+                const isIncidentManager = currentUser.belongToUserGroup(userGroup.id);
```

Comment:
if my understanding is correct, this userGroup will be same throughout the app (wont be different for different evnt trackers) So can you move this code to GetConfigurationsUseCase so that it is called only once on app load? You can add isIncidentManager to AppContextState.currentUser and save the value there and it will be available throughout the code.